transformations/chinese_simplified_traditional_perturbation/README.md

```diff
@@ -0,0 +1,24 @@
+# Chinese Simplified （简体）and Traditional （繁体）Perturbation 🦎  + ⌨️ → 🐍
+This perturbation adds noise to all types of text sources containing Chinese words and characters (sentence, paragraph, etc.) by changing the words and characters between Simplified and Traditional Chinese.
+
+Author name: Timothy Sum Hon Mun
+Author email: timothy22000@gmail.com
+
+## What type of a transformation is this?
+This transformation perturbes Chinese input text to test robustness. Chinese words or characters that are in Simplified Chinese are picked at random will be replaced with words and characters in
+Traditional Chinese. This transformation can also be performed in the opposite direction. 
```

Comment:
Is it `can be`? Is it implemented here? Do you need to change a parameter or is it done automatically? 
Could you add more information here?

Reply:
You'll have to change the `converter_config` argument. I'll update it in the README.md